compio/benches/fs.rs

```diff
@@ -443,6 +554,37 @@ fn write_all_tokio(b: &mut Bencher, (path, content): &(&Path, &[u8])) {
     })
 }
 
+fn write_all_tokio_std(b: &mut Bencher, (path, content): &(&Path, &[u8])) {
+    let runtime = tokio::runtime::Builder::new_current_thread()
+        .enable_all()
+        .build()
+        .unwrap();
+    b.to_async(&runtime).iter_custom(|iter| async move {
+        let content = Arc::new(content.to_vec());
+        let mut file = std::fs::File::create(path).unwrap();
+
+        let start = Instant::now();
+        for _i in 0..iter {
+            let content = content.clone();
+            file = tokio::task::spawn_blocking(move || {
+                file.seek(SeekFrom::Start(0)).unwrap();
+                let mut write_len = 0;
+                let total_len = content.len();
+                while write_len < total_len {
```

Comment:
In this function, all write operations are performed inside the same `spawn_blocking` closure. It should be as fast as `write_all_std`.